<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_3"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSubtract()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotDivide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMultiplicate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSum()</slot>
  <slot>slotSubtract()</slot>
  <slot>slotDivide()</slot>
  <slot>slotMultiplicate()</slot>
 </slots>
</ui>
